data_tools/buildhub/utils.py

```diff
@@ -0,0 +1,93 @@
+from os.path import basename
+
+
+def build_record_from_url(url):
+    record = {}
+    parts = basename(url).split('.')
+    url_parts = url.split('/')
+
+    product_name, major_version = parts[0].split('-')
```

Comment:
I'm not super confortable with the fact that parse the product from url, since the product is something we have in the code in the `PRODUCTS` tuple.